### PR TITLE
Address late review for Android TV Remote media_player platform

### PR DESCRIPTION
**Pull request by Drafteed**
## Proposed change

Tweaks from late review: https://github.com/home-assistant/home-assistant.io/pull/27059

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Review by frenck — CHANGES_REQUESTED** [review]
The problem, is that the original (and this PR) is changing parts of the current stable documentation; while targeting the `next` branch.

We need to revert parts of the referenced PR that don't belong in the next branch.

**Comment by tronikos** [comment]
There is also going to be some conflicts with my #27273 when next becomes current.
@frenck is it possible to have the owner get notified for documentation changes?
@Drafteed regarding your previous #27059 I'd strongly prefer we keep the source of truth for supported keys in the underlying library and continue linking to TvKeys.txt. I want to avoid having to keep two lists in sync. Feel free to send me a PR updating the TvKeys.txt if needed.

I recommend merging my #27273 and then Drafteed works on incorporating changes in this PR and in the previous #27059 in the current branch. Then maybe rebase next with current and finally incorporate changes from #27059 and this PR in the next branch.

**Comment by frenck** [comment]
> @frenck is it possible to have the owner get notified for documentation changes?

In this case, that already happened, the original PR was even linked to the core PR which you have been marked codeowner for (and the bot pinged you for).

> I recommend merging my https://github.com/home-assistant/home-assistant.io/pull/27273 and then Drafteed works on incorporating changes in this PR

I don't agree. I recommend al none `next` related changes #27059 get reverted in this PR. They didn't belong in that PR to begin with.

**Comment by tronikos** [comment]
Yes I reviewed the core PR and I saw the documentation PR because it was linked there but I wasn't explicitly asked to review it so I never had a chance to leave comments. I mistakenly assumed it was a draft. I would have also pointed out to move unrelated changes to the current branch. There is no way to have owners get notified for documentation changes? Or even require their approval like core?

Reverting unrelated changes, reverting the whole #27059, or rebasing next with current after updating current and reapplying the changes will achieve the same effect. I'll let you chose what will make your life easier when you make next to current in a few weeks.

**Comment by frenck** [comment]
> There is no way to have owners get notified for documentation changes?

That happens, in this case, there was no need, as this was part of the parent PR you have been notified about.

> Or even require their approval like core?

We don't have that policy on documentation as of yet. The PR has been handled correctly in terms of process, however, something slipped in. That happens, mistakes happen. I understand when it happens it raises al kinds of questions, but let's keep all that out of this PR review.

Discussions and questions like these distract from the actual thing at hand: Correcting the mistakenly merged changes.

> I'll let you chose what will make your life easier when you make next to current in a few weeks.

For me, it doesn't matter, if this isn't handled, I will revert incorrect PRs eventually to get them out of the way.

../Frenck

**Comment by Drafteed** [comment]
> Yes I reviewed the core PR and I saw the documentation PR because it was linked there but I wasn't explicitly asked to review it so I never had a chance to leave comments. I mistakenly assumed it was a draft. I would have also pointed out to move unrelated changes to the current branch. There is no way to have owners get notified for documentation changes? Or even require their approval like core?

I even asked you in core PR if you looked at the PR documentation 3 days ago: https://github.com/home-assistant/core/pull/91677#discussion_r1186208425
I didn't know you were trying to change the current branch in #27273

@frenck I have returned in this PR all parts changed in the previous PR, except `Media Player` section and metas. Now the documentation looks unstructured and needs some love.

File: source/_integrations/androidtv_remote.markdown
Comment (tronikos) [inline source/_integrations/androidtv_remote.markdown]: Most Android TV devices don't come with a built-in browser.

File: source/_integrations/androidtv_remote.markdown
Comment (tronikos) [inline source/_integrations/androidtv_remote.markdown:26]: (This is for line 29, for some reason I cannot leave a comment there)
Use relative paths to link to cast and universal integrations. That way they work correctly for example on https://rc.home-assistant.io/

File: source/_integrations/androidtv_remote.markdown
Comment (tronikos) [inline source/_integrations/androidtv_remote.markdown]: I think you want to rename this section to Remote now

File: source/_integrations/androidtv_remote.markdown
Comment (tronikos) [inline source/_integrations/androidtv_remote.markdown:26]: Consider moving the Media player section after the Remote section. That way the reader knows what deep links are. Once this documentation makes it to current you can restructure the documentation to add common sections that belong to both media player and remote sections and reorder them if needed.

**Review by tronikos — CHANGES_REQUESTED** [review]

File: source/_integrations/androidtv_remote.markdown
Comment (tronikos) [inline source/_integrations/androidtv_remote.markdown]: Change this to ###? I think this would be better to be under the Remote section.

File: source/_integrations/androidtv_remote.markdown
Comment (tronikos) [inline source/_integrations/androidtv_remote.markdown]: Same here. I think ### would be better.

**Review by tronikos — CHANGES_REQUESTED** [review]

File: source/_integrations/androidtv_remote.markdown
Comment (Drafteed) [reply source/_integrations/androidtv_remote.markdown]: I just reverted the changes before #27059, since it was marked as unrelated.

File: source/_integrations/androidtv_remote.markdown
Comment (Drafteed) [reply source/_integrations/androidtv_remote.markdown]: Same as https://github.com/home-assistant/home-assistant.io/pull/27308#discussion_r1188366143
I just reverted the changes before https://github.com/home-assistant/home-assistant.io/pull/27059.

File: source/_integrations/androidtv_remote.markdown
Comment (Drafteed) [reply source/_integrations/androidtv_remote.markdown]: Same as https://github.com/home-assistant/home-assistant.io/pull/27308#discussion_r1188366143
I just reverted the changes before https://github.com/home-assistant/home-assistant.io/pull/27059.

File: source/_integrations/androidtv_remote.markdown
Comment (Drafteed) [reply source/_integrations/androidtv_remote.markdown:26]: Done

**Comment by Drafteed** [comment]
To avoid confusion, here is a link to the diff between the current branch:
https://github.com/home-assistant/home-assistant.io/compare/current...Drafteed:home-assistant.io:androidtv-remote-follow-up#diff-89013b20875ba4766abf1eed6d84c7b21f26f7d53e7f587913e48bcc3cd3ffb2

I've reverted all changes before #27059 and left only the media player section + new metadata.

File: source/_integrations/androidtv_remote.markdown
Comment (tronikos) [reply source/_integrations/androidtv_remote.markdown]: I wouldn't call this unrelated. The old documentation was structured for implementing a single platform. Since now it's 2 platforms you need to structure it accordingly and have 2 sections with subsections. You should just try to avoid making changes that also apply to the current branch. Those are unrelated to the addition of the new platform.

Comment (Drafteed) [reply source/_integrations/androidtv_remote.markdown]: Done

**Review by tronikos — APPROVED** [review]

**Comment by tronikos** [comment]
Close this now that the changes made it to current?

**Comment by Drafteed** [comment]
Closing this in favor of #27273, since `next` merged into `current`.

**Pull request closed without merging**
